client/controllers/curatorInbox/curatorSourceDetails.coffee

```diff
@@ -162,7 +148,10 @@ Template.curatorSourceDetails.events
   'click span.annotation': (event, instance) ->
     incidentId = $(event.currentTarget).data('incident-id')
```

Comment:
@Broham Yes and no. I think the previous case was also storing wether the IR was accepted or not in the DOM (determined by whether the element has a class, not in the data layer). I'm not sure we can avoid storing the ID in the DOM here because the context of the annotation is the source, not the IR. (But, since we fetch the collection while parsing the text to add the annotations, when the collection changes, the state of the annotations are reactively updated. So the state is still in the data layer). I think the main reason we are storing the ID here is so we can easily access the spans with jQuery (scrollTo) and if we need to access the IR on click (e.g. editing the IR).